aries_cloudagent/protocols/connections/v1_0/manager.py

```diff
@@ -296,7 +296,8 @@ async def create_request(
 
         # Create connection request message
         if not my_endpoint:
-            my_endpoints = [self.context.settings.get("default_endpoint")]
+            my_endpoints = []
+            my_endpoints.extend(self.context.settings.get("default_endpoint", []))
```

Comment:
That doesn't seem right, `default_endpoint` is a string

Reply:
You're right, will fix